.pre-commit-config.yaml

```diff
@@ -13,6 +13,7 @@ repos:
       - id: check-toml
       - id: check-xml
       - id: debug-statements
+        files: ^(?!{{cookiecutter.project_slug}}\/settings\.py).*$
```

Comment:
it stinks me a bit

Reply:
@trottomv that code was not part of my proposal, so I ended up removing it.